plugins/ssr/ssrparser.mlg

```diff
@@ -79,8 +80,39 @@ let ssrtac_entry name = {
   mltac_index = 0;
 }
 
-let register_ssrtac name f =
-  Tacenv.register_ml_tactic (ssrtac_name name) [|f|]
+let cache_tactic_notation (_, (key, body, parule)) =
+  Tacenv.register_alias key body;
+  Pptactic.declare_notation_tactic_pprule key parule
+
+type tactic_grammar_obj = KerName.t * Tacenv.alias_tactic * Pptactic.pp_tactic
+
+let inSsrGrammar : tactic_grammar_obj -> obj =
+  declare_object {(default_object "SsrGrammar") with
+                  load_function = (fun _ -> cache_tactic_notation);
+                  cache_function = cache_tactic_notation;
+                  classify_function = (fun x -> Keep x)}
+
+let path = MPfile (DirPath.make @@ List.map Id.of_string ["Coq"; "ssr"; "ssreflect"])
```

Comment:
The list needs to be reversed. It probably doesn't matter in practice (somebody calling their module ssreflect.ssr.Coq probably deserves what happens), but just for clarity it should be fixed.